internal/pkg/cli/run_local.go

```diff
@@ -815,12 +831,48 @@ func (h *hostDiscoverer) Hosts(ctx context.Context) ([]orchestrator.Host, error)
 			for _, alias := range sc.ClientAliases {
 				hosts = append(hosts, orchestrator.Host{
 					Name: aws.StringValue(alias.DnsName),
-					Port: strconv.Itoa(int(aws.Int64Value(alias.Port))),
+					Port: uint16(aws.Int64Value(alias.Port)),
 				})
 			}
 		}
 	}
 
+	// get rds instances
+	dbs, err := h.rg.GetResourcesByTags(resourcegroups.ResourceTypeRDSInstance, map[string]string{
+		deploy.AppTagKey: h.app,
+		deploy.EnvTagKey: h.env,
+	})
+	switch {
+	case err != nil:
+		return nil, fmt.Errorf("get tagged rds instances: %w", err)
+	case len(dbs) == 0:
+		return hosts, nil
+	}
+
+	filter := &rds.Filter{
+		Name: aws.String("db-instance-id"),
+	}
+	for i := range dbs {
+		filter.Values = append(filter.Values, aws.String(dbs[i].ARN))
+	}
+
+	err = h.rds.DescribeDBInstancesPagesWithContext(ctx, &rds.DescribeDBInstancesInput{
```

Comment:
Maybe comment that we're calling the API directly here? I think in most cases in the codebase we've wrapped the calls in the `aws` package and this use case makes sense, but I think it's a unique enough case to point out to avoid confusion.